src/library/OverflowContainer/__tests__/OverflowContainerWithShadows.spec.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import React from 'react';
+import { mineralTheme } from '../../themes';
+import { getProcessedComponentThemeKeys } from '../../themes/processComponentTheme';
+import OverflowContainerWithShadows, {
+  componentTheme
+} from '../OverflowContainerWithShadows';
+import { mountInThemeProvider } from '../../../../utils/enzymeUtils';
+import testThemeOverrides from '../../../../utils/testThemeOverrides';
+
+const mountOverflowContainerWithShadows = (props = {}) =>
+  mountInThemeProvider(<OverflowContainerWithShadows {...props} />);
+
+describe('OverflowContainerWithShadows', () => {
+  it('renders', () => {
+    const [, overflowContainer] = mountOverflowContainerWithShadows();
+
+    expect(overflowContainer.exists()).toEqual(true);
+  });
+
+  // TODO: I suspect that we cannot use findDOMNode inside testThemeOverrides,
+  //       which would prevent either shadow variable from ever applying. If so,
+  //       we don't need the fixed heights below.
+  describe('theme overrides', () => {
+    testThemeOverrides(
+      <OverflowContainerWithShadows style={{ height: 100 }}>
+        <div style={{ height: 1000 }} />
+      </OverflowContainerWithShadows>,
+      getProcessedComponentThemeKeys(componentTheme(mineralTheme), {
+        excludeKeys: [
+          'OverflowContainerWithShadows_boxShadowBottom',
+          'OverflowContainerWithShadows_boxShadowTop'
```

Comment:
Since these are the only unique component theme variables for OverflowContainerWithShadows, do we need to test the component theme here?

Reply:
That's a good catch. In this case, I think it's still helpful, as it confirms that we've correctly mapped the OverflowContainer component theme onto this one.